web-vitals.md

```diff
@@ -0,0 +1,23 @@
+---
+page: web-vitals
```

Comment:
@dainelmawer 

Are we sure we want to be as specific as calling this new page Web Vitals? Should we consider calling it Performance with a current focus on the definition and recommendations pertaining to Web Vitals? Could other performance related resources, tools, best practices come into play here? We could briefly mention performance relating to specific sections in the BP doc are mentioned elsewhere, while general performance best practices regarding perfromance are defined here. 

Something to ponder. I don't want to block the update because of this thought. If this has already been thought about / discussed - disregard my comment and move forward.

Reply:
@joesnellpdx appreciate your opinion here. Here was my thinking behind it - when I look online across sites that are listing performance improvements specifically around Web Vitals - the URL all kind of centers around the "web-vitals" buzzword. I wanted to make sure that it's absolutely clear that 10up supports Web Vitals in a direct manner. Having a nav item as Web Vitals I think will help with this and we can offer targeted advice around fixing specific web vitals - rather than the general catch-all "performance" term that is already listed under each discipline/language.

I think we can iterate on this approach and clean it up at some point but with the algorithm update approaching fast, I wanted to make sure we were being as transparent as possible.

Reply:
I totally get what you are saying. Per our conversation today, I think the plan is to go with a Performance page and CWV as part of it... but I still like the idea of a direct web vitals url. I'll think about it some more. Maybe after the fact, we could do a post on our external blog and link to our best practices? Just an idea. 

@dainelmawer

Reply:
Roger that - I think an external blog post would be a great idea.